### ISSUE TITLE
visibleInSelection not working

### ISSUE THREAD

**Issue by philliptanner** [rstaib/jquery-bootgrid#202]
It appears the data-visibleInSelection="false" is not mapping through to the dropdown filter.

Debugging the script v1.3.1 the line visibleInSelection: !(data.visibleInSelection === false) the property is undefined and it is coming through all lower case as data.visibleinselection.

**Comment by steferson** [comment]
Thank you!
I had to change  !(data.visibleInSelection === false) to  !(data.visibleinselection === false)
on my script to get this working.

**Comment by ckiklas** [comment]
I'm having the same issue, and changing what you did above didn't make any difference. Columns with the visibleInSelection marked false still show up in the dropdown for selection.

**Comment by steferson** [comment]
I changed line 129 in Bootgrid v1.3.1. Was that the same line that you changed?
And did you also change the corresponding part on the min version of the file?

This is how my table header ended up:
 th data-column-id="id" data-visible="false" data-visibleinselection="false" data-type="numeric" data-identifier="true"

**Comment by ckiklas** [comment]
Yeah, that's where I changed it. I then found another issue on here that said to use visible-in-selection and that fixed it for me.

**Comment by renerlemes** [comment]
You need to use data-visible-in-selection="false".

It's working for me
